### PR TITLE
Add Optimus to integration test

### PR DESCRIPTION
**Pull request by samanehsan**
This PR adds a test for the Optimus pipeline, which should be run as a part of the integration tests after the pipeline is running in the DCP.

### CODE REVIEW

**Comment by samanehsan** [comment]
The test for Optimus is passing: https://allspark.dev.data.humancellatlas.org/HumanCellAtlas/dcp/-/jobs/24852

File: .gitlab-ci.yml [hunk 1]
```diff
@@ -6,6 +6,7 @@ stages:
   - 100_cell_test
 
 before_script:
+  - export CI_COMMIT_REF_NAME=integration
```
Comment (rexwangcc) [inline .gitlab-ci.yml]: Please remember to remove those testing code before merging! :octocat:

File: tests/integration/test_end_to_end_dcp.py [hunk 2]
```diff
@@ -163,5 +163,73 @@ def test_10x_run(self):
...
+        re.compile('^.+zarr!expression_matrix!gene_metadata_numeric!\.zarray$'),
+        re.compile('^.+zarr!expression_matrix!gene_metadata_numeric!0\.0$'),
+        re.compile('^.+zarr!expression_matrix!gene_metadata_numeric_name!\.zarray$'),
+        re.compile('^.+zarr!expression_matrix!gene_metadata_numeric_name!0$')
+    ]
+
+    def test_optimus_run(self):
+        runner = DatasetRunner(deployment=self.deployment)
+
+        with Timeout(210 * 60) as to:  # timeout after 3 hours
```
Comment (ApChagi) [inline tests/integration/test_end_to_end_dcp.py]: Are the units seconds here? Makes 3.5 hours if so

Comment (samanehsan) [reply tests/integration/test_end_to_end_dcp.py]: Yes! I forgot to change the comment last time

File: tests/dataset_runner.py [hunk 3]
```diff
@@ -99,7 +99,7 @@ def run(self, dataset_fixture, run_name_prefix="test"):
                 self.wait_for_primary_bundles()
                 self.wait_for_analysis_workflows()
                 self.wait_for_secondary_bundles()
-            if self.dataset.name == "Smart-seq2":
+            if self.dataset.name == "Smart-seq2" or self.dataset.name == "optimus":
```
Comment (rexwangcc) [inline tests/dataset_runner.py]: Not sure if we want to disable matrix for optimus too, @parthshahva ?

File: tests/integration/test_end_to_end_dcp.py [hunk 4]
```diff
@@ -163,5 +163,73 @@ def test_10x_run(self):
             raise TimeoutError("test timed out")
 
 
+class TestOptimusRun(TestEndToEndDCP):
```
Comment (rexwangcc) [inline tests/integration/test_end_to_end_dcp.py]: So we are not removing the cellranger runner, instead, we don't run it?

File: tests/integration/test_end_to_end_dcp.py (hunk 2 above)
Comment (rexwangcc) [reply tests/integration/test_end_to_end_dcp.py]: It'd be great if the comments can make it clear that this number is consistent with what we set in the Gitlab runner settings.

File: tests/dataset_runner.py (hunk 3 above)
Comment (samanehsan) [reply tests/dataset_runner.py]: I just noticed it's commented out on the `integration` branch so I will do the same here

File: tests/integration/test_end_to_end_dcp.py (hunk 4 above)
Comment (samanehsan) [reply tests/integration/test_end_to_end_dcp.py]: I'll actually remove the test for cellranger as well just so there isn't any confusion there

File: tests/dataset_runner.py (hunk 3 above)
Comment (rexwangcc) [reply tests/dataset_runner.py]: FYI: https://humancellatlas.slack.com/archives/G9XD6L0AD/p1553277875269900

**Pull request merged**